### ISSUE TITLE
ikea TRÅDFRI Wireless dimmer

### ISSUE THREAD

**Issue by vincentserpoul** [mozilla-iot/zigbee-adapter#159]
item: https://www.ikea.com/us/en/p/tradfri-wireless-dimmer-white-20347830/

considering the price and the functionality, it would be nice to make it work properly.
I had quite some trouble to pair it, but I finally managed to.
It is detected with on/off and a level, but unfortunately, it doesn't change when the dimmer is turned.
Is there any walkthrough to help debug this?

**Comment by dhylands** [comment]
IIRC that particular dimmer assumes that its controlling a light and issues a read request to the light.
The conbee dongle emulates a light just so that the IKEA dimmers can be supported.

I'm prety sure that I saw the read request come in when I had frame debugging enabled, but I haven't written support to deal with that yet.

**Comment by flatsiedatsie** [comment]
I think IKEA has stopped selling this device.

**Comment by vincentserpoul** [comment]
they still do: https://www.ikea.com/jp/en/p/tradfri-dimming-kit-warm-white-white-30349937/ (as an example)

**Comment by flatsiedatsie** [comment]
Ah, glad to hear it. It's no longer available on the Dutch website, I couldn't find it at my local IKEA, and the lady I asked about it said she didn't know that device I meant.

**Comment by flatsiedatsie** [comment]
I checked again, and they definitely don't. They do have something else that they call the "dimmer kit" here now. It comes with a very different button:

https://www.ikea.com/nl/en/p/tradfri-wireless-dimmer-white-70408595/

During that search I also noticed that IKEA does have something that does get close to the round dimmer: the Symfonisk Remote. 

https://www.ikea.com/nl/en/p/symfonisk-sound-remote-white-60370480/

...which The Verge loves:
https://www.theverge.com/2019/10/1/20892984/ikea-sound-controller-review-symfonisk

I couldn't find it on the IKEA website, but The Verge claims it uses Zigbee.
